### PR TITLE
Move strings here from lib-cove and ensure everything is translated

### PR DESCRIPTION
**Pull request by Bjwebb**
Corresponding lib-cove PR is https://github.com/OpenDataServices/lib-cove/pull/68

We move strings here, because it makes them easier to translate with Django's translation machinery, and means we can remove the Django dependency of lib-cove.

### CODE REVIEW

**Comment by Bjwebb** [comment]
@jpmckinney I still need to ensure tests pass on CI, but this is ready for a look through (I can't assign you as reviewer). Still to come are PRs on cove-oc4ids and flatten-tool.

**Comment by jpmckinney** [comment]
@Bjwebb I just skimmed, and I trust that it works :), but let me know if there were some specific angle you wanted reviewed.

File: cove/templatetags/cove_tags.py [hunk 1]
```diff
@@ -56,3 +62,87 @@ def take_or_sample(population, k):
...
+    "integer": _("{}<code>{}</code> is not a integer. Check that the value {} doesn’t contain decimal points or any characters other than 0-9. Integer values should not be in quotes. "),
+    "number": _("{}<code>{}</code> is not a number. Check that the value {} doesn’t contain any characters other than 0-9 and dot (<code>.</code>). Number values should not be in quotes. "),
+    "object": _("{}<code>{}</code> is not a JSON object"),
+    "array": _("{}<code>{}</code> is not a JSON array"),
+}
+
+
+@register.filter(name='html_error_msg')
+def html_error_msg(error):
+
```
Comment (michaelwood) [inline cove/templatetags/cove_tags.py:83]: Might be nice to have a doc string description here given that it is a library function.

File: cove/templatetags/cove_tags.py [hunk 2]
```diff
@@ -145,4 +145,16 @@ def html_error_msg(error):
             error.get("instance"),
         )
 
+    if e_validator == "minProperties":
+        return _("{} does not have enough properties").format(e_instance)
+
+
+    if error.get("error_id"):
```
Comment (michaelwood) [inline cove/templatetags/cove_tags.py:152]: might be a bit more pythonic to use try and catch the keyerror?

**Comment by michaelwood** [comment]
Can't see any issues. Only minor comments which you can ignore if you want.

**Review by michaelwood — DISMISSED** [review]
(once CI is happy)

File: cove/templatetags/cove_tags.py (hunk 2 above)
Comment (Bjwebb) [reply cove/templatetags/cove_tags.py:152]: This also handles the case where the value exists, but is None (which does happen for this).

**Review by michaelwood — APPROVED** [review]

**Pull request merged**